spec/services/backfill_notifications_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+RSpec.describe BackfillNotifications do
```

Comment:
I would suggest we delete these sometime in the future, since this script should only be used once and there's no reason to test it forever.

Reply:
👍